Cargo.toml

```diff
@@ -120,7 +120,7 @@ exitcode = "1.1.2"
 snafu = "0.4.3"
 url = "1.7"
 base64 = "0.10.1"
-shiplift = "0.5"
+shiplift = { git = "https://github.com/LucioFranco/shiplift", branch = "lucio/fix-encrypted-tcp" }
```

Comment:
This uses my fork which is related to this PR https://github.com/softprops/shiplift/pull/193. I know the author so I will but them about getting a release but for now we can work off of my fork.

Reply:
In https://github.com/timberio/vector/pull/787/commits/ed82e033d7978cda9ff640a9390bae3287fde2a3 we now use the `timber` branch of my fork that contains a feature flag fix and rust 2018 idioms `cargo fix` pass.

Reply:
Great!

We should probably make an issue to remember to revert this to proper shiplift version.